shadows/framework/src/main/java/org/robolectric/shadows/ShadowChoreographer.java

```diff
@@ -256,7 +255,7 @@ void doFrame(
     void __constructor__(Looper looper, int vsyncSource);
 
     @Accessor("mFrameData")
-    FrameData getFrameData();
+    /*android.view.Choreographer$FrameData*/ Object getFrameData();
```

Comment:
Would it make sense to use `@ClassName` here?
```suggestion
    @ClassName("android.view.Choreographer$FrameData") Object getFrameData();
```